processing/src/main/java/io/druid/query/MetricsEmittingExecutorService.java

```diff
@@ -66,6 +71,7 @@ public void emitMetrics(ServiceEmitter emitter, ServiceMetricEvent.Builder metri
   {
     if (delegate instanceof PrioritizedExecutorService) {
       emitter.emit(metricBuilder.build("segment/scan/pending", ((PrioritizedExecutorService) delegate).getQueueSize()));
+      emitter.emit(metricBuilder.build("merge/buffers/size", ((DefaultBlockingPool) mergeBufferPool).getPoolSize()));
```

Comment:
Suggest calling this `query/merge/buffersUsed`. Presumably, the other metric (merge buffer wait time) would be something like `query/merge/wait`.